### DISCUSSION TITLE
specFileRetriesDeferred default value in wdio v7

### DISCUSSION THREAD

**Discussion by SumanDas94** [webdriverio/webdriverio#13997] [Q&A]
Need some help to understand what is the default value for `specFileRetriesDeferred` in wdio v7.x.x 
Is this value being `true` refers to failed specs being deferred for retries or does that mean the value should be `false` ?

Default values seem to be different [here](https://webdriver.io/docs/configurationfile/) and [here](https://webdriver.io/docs/configuration/#specfileretriesdeferred)

**Comment by christian-bromann** [comment]
> in wdio v7.x.x

I really recommend to update to v9, we have deprecated v7.

We indeed have some confusing statements what the default value is here. I raised [a PR](https://github.com/webdriverio/webdriverio/pull/14001) to clean this up. The default value should be `false` afaik given the default value that is being assigned here: https://github.com/webdriverio/webdriverio/blob/d2d312a0400ed31b09a58eeada5637ed5d18c478/packages/wdio-config/src/constants.ts#L36
